tests/manager/manager-executeWriteCommand_error-002.phpt

```diff
@@ -0,0 +1,33 @@
+--TEST--
+MongoDB\Driver\Manager::executeWriteCommand() throws CommandException for invalid writeConcern
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php NEEDS('REPLICASET'); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$manager = new MongoDB\Driver\Manager(REPLICASET);
+
+$command = new MongoDB\Driver\Command([
+    'findAndModify' => COLLECTION_NAME,
+    'query' => ['_id' => 'foo'],
+    'update' => ['foo' => ['bar']],
+    'upsert' => true,
+    'new' => true,
+]);
+
+try {
+    $manager->executeWriteCommand(DATABASE_NAME, $command, ['writeConcern' => new MongoDB\Driver\WriteConcern("undefined")]);
+} catch (MongoDB\Driver\Exception\CommandException $e) {
+    printf("%s(%d): %s\n", get_class($e), $e->getCode(), $e->getMessage());
+    assert($e->getMessage() === "Write Concern error: " . $e->getResultDocument()->writeConcernError->errmsg);
+    assert($e->getCode() === $e->getResultDocument()->writeConcernError->code);
```

Comment:
I use "assert" loosely when referring to the PHPC tests; sorry if that came across as a suggestion for using [`assert()`](http://php.net/assert). If you read further down the page, you'll see that `assert()` behavior can vary wildly by INI settings, which makes it less than ideal for testing purposes. This is why `phpt` files (in PHP core and our extension) mostly rely on dumping and comparing output.

Rather than change `assert()` to `var_dump()` and add "bool(true)" to the expected output, let's just `var_dump($e->getResultDocument())` here as you do in the next test.

Since the `printf()` includes the code and message and `var_dump()` covers the result document itself, these comparisons are redundant. I don't think I was thinking about `var_dump($e->getResultDocument())` when I suggested assertions earlier today :)